front/lib/error.ts

```diff
@@ -28,7 +28,11 @@ export type DustErrorCode =
   | "remote_server_not_found"
   | "internal_server_not_found"
   // Space errors
-  | "space_already_exists";
+  | "space_already_exists"
+  // Excel errors
+  | "invalid_upsert_args"
+  | "invalid_content_error"
```

Comment:
Let's just keep invalid_content_error and move it under table (remove reference to Excel here too specific)